Gemfile

```diff
@@ -17,13 +18,15 @@ gem 'jbuilder', '~> 2.10'
 gem 'oj', '~> 3.9', '>= 3.9.2'
 gem 'pagy', '~> 4.0'
 gem 'pg', '~> 1.1', '>= 1.1.4'
+# gem 'pg', '~> 1.5', '>= 1.5.3'
 gem 'puma', '~> 5.6'
 gem 'pundit', '~> 2.1'
 gem 'rack-cors', '~> 1.0', '>= 1.0.6'
 gem 'sass-rails', '~> 6.0.0'
 gem 'sendgrid', '~> 1.2.4'
 gem 'sprockets', '~> 3.7.2'
 gem 'yaaf', '~> 2.2'
+gem 'wdm', '>= 0.1.0'
```

Comment:
I think this gem can be moved to the `development` group, since its used for development experience only and not needed on a production environment, wdyt?